core/CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ set(checks_SRC
 list(APPEND main_SRC ${checks_SRC})
 
 set(include_SRC
-    "include/aes_gcm.h"
+    "include/aes_gcm_dev.h"
```

Comment:
I'm not super familiar with cmake syntax. This is OK in the main_SRC list and doesn't need to have a target check?

Reply:
This is fine. `include_SRC` lets the build system know where to find the headers to include for the source files that include them.